### ISSUE TITLE
EventDispatcher Symfony 4.3

### ISSUE THREAD

**Issue by JulienMoreau** [DataDog/dd-trace-php#461]
Since 4.3, the event name is't required in dispatch method.

https://symfony.com/blog/new-in-symfony-4-3-simpler-event-dispatching

Can you modify \DDTrace\Integrations\Symfony\V4\SymfonyBundle:147 for detect if $args[0] is an object ?

Like :

```
if (is_object($args[0])) {
    if (isset($args[1])) {
         $eventName = $args[1];
    } else {
         $eventName = get_class($args[0]);
    }
} else {
     $eventName = $args[0];
}
$scope = GlobalTracer::get()->startIntegrationScopeAndSpan(
    SymfonyIntegration::getInstance(),
    'symfony.' . $eventName
);
```

Thanks

**Comment by mglinski** [comment]
I am also having this issue after upgrading an API to symfony 4.3:

![](https://cl.ly/b5c5271356a8/Image%202019-06-06%20at%2012.01.15%20PM.png)

**Comment by SammyK** [comment]
Thanks for the report @JulienMoreau and for the screenshot @mglinski!

What version of the PHP tracer is installed? This issue was patched in #466. The latest tracer version ([as of writing `0.27.2`](https://github.com/DataDog/dd-trace-php/releases/tag/0.27.2)) contains a patch for this issue. Let us know if the issue persists after upgrading! :)

**Comment by mglinski** [comment]
Yea, I was a bout to submit a patch for this but saw that it was already fixed.

I commented here after I asked our host to upgrade the tracer install and still had issues, but after reviewing the files on the server, the upgrade apparently did not happen.

I am having them retry the upgrade and will report if my issue is fixed once they complete the upgrade correctly.
